base/bool.jl

```diff
@@ -83,6 +83,9 @@ returning [`missing`](@ref) if one of the arguments is `missing`.
 The infix operation `a ⊼ b` is a synonym for `nand(a,b)`, and
 `⊼` can be typed by tab-completing `\\nand` or `\\barwedge` in the Julia REPL.
 
+!!! compat "Julia 1.7"
+    Support for `nand` and `nor` requires at least Julia 1.7.
```

Comment:
You don't need to document the compatibility of `nor` method in the docstring of `nand` method.

```suggestion
    This method requires at least Julia 1.7.
```

Reply:
Although something like ``See also [`xor`](@ref), [`nor`](@ref).`` may not be a bad idea.